src/_animate.scss

```diff
@@ -1,3 +1,23 @@
+@function duration($transition) {
+    @if $transition == "open" {
+        @return 250ms;
+    } @else if $transition == "close" {
+        @return 200ms;
+    }
+
+    @if $transition == "expand" {
+        @return 300ms;
+    } @else if $transition == "collapse" {
+        @return 250ms;
+    }
+
+    @if $transition == "element" {
+        @return 100ms;
+    }
+}
+
+$easing: cubic-bezier(0.4, 0, 0.2, 1);
```

Comment:
GTK doesn't seem to love this… it seems like it is less smooth than the `ease-out` in the scrollbar, especially—I see it animating the last frame much slower than the previous easing. Not sure if it matters that much, but it's gonna bother me. :stuck_out_tongue: